acquire/utils.py

```diff
@@ -368,10 +368,13 @@ def persist_execution_report(path: Path, report_data: dict) -> Path:
         f.write(json.dumps(report_data, sort_keys=True, indent=4))
 
 
-SYSVOL_SUBST = re.compile(r"^(/\?\?/)?[cC]:")
+DEVICE_SUBST = re.compile(r"^(/\?\?/)")
+SYSVOL_SUBST = re.compile(r"^/?sysvol(?=/)")
 
 
-def normalize_path(target: Target, path: Path, resolve: bool = False, lower_case: bool = True) -> str:
+def normalize_path(
+    target: Target, path: Path, *, sysvol: Optional[str] = None, resolve: bool = False, lower_case: bool = True
```

Comment:
There is some ugly dependency between `normalize_path()` and `serialize_path()`. It seems the latter sometimes depends on paths already being put through the former.

Currently there are again duplicates being collected as `serialize_path()` does not know about the maping between sysvol and the drive letter. As `sysvol` is optional in `normalize_path` things like `from_user_home()` emit both sysvol and drive letter paths, which the dedup logic (using `serialize_path()`) does not catch.

Ideally we rewrite acquire to use single canonical paths, but maybe we can get away with not making sysvol optional, so it is replaced everywhere `normalize_path()` is used.

Reply:
Opted for the latter for now.

Reply:
Unfortunately it does not fix the de-duplication issue with `serialize_path()`, but for this PR I think that's fine as we've planned to tackle the de-dup issue separately.